lib/python/qmk/cli/c2json.py

```diff
@@ -21,19 +21,14 @@ def c2json(cli):
 
     This command uses the `qmk.keymap` module to generate a keymap.json from a keymap.c file. The generated keymap is written to stdout, or to a file if -o is provided.
     """
-    cli.args.filename = qmk.path.normpath(cli.args.filename)
-
-    # Error checking
-    if not cli.args.filename.exists():
-        cli.log.error('C file does not exist!')
-        cli.print_usage()
-        exit(1)
-
-    if str(cli.args.filename) == '-':
-        # TODO(skullydazed/anyone): Read file contents from STDIN
-        cli.log.error('Reading from STDIN is not (yet) supported.')
-        cli.print_usage()
-        exit(1)
+    if cli.args.filename != '-':
+        cli.args.filename = qmk.path.normpath(cli.args.filename)
+
+        # Error checking
+        if not cli.args.filename.exists():
+            cli.log.error('C file does not exist!')
+            cli.print_usage()
+            exit(1)
```

Comment:
I'm pretty sure you can just `return False` here, and MILC will set the return code as necessary.

Reply:
Definitely.
It can be changed in this PR or I have one in the works that's based on this one which updates/fixes the behaviour of `c2json` and changes this to `return False`.